shared/getDeviceOnDash/artik710.md

```diff
@@ -0,0 +1,32 @@
+## Setting Up Your Device
+
+In order to get the resinOS up and running on our {{ $device.name }} we need to first set it up to boot from our {{ $device.bootMedia }} rather than its 
+internal [eMMC memory][emmc-link]. To do this we need to set the `SW402` dip switch to position `1:off`, `2:off`, `3:off`, `4:on`, and insert the your {{ $device.bootMedia }}
+into the board, **hint:** The SD card slot can be found on the underside of the board near the Audio Jack.
+
+__Note:__ This resinOS will completely write over the existing eMMC.
+
+<!-- <img src="/img/artik5/artik5-dev-kit.png" width="80%"> -->
+If you selected wifi as your connectivity type, remember to attach the wifi antennas before booting the board.
+
+We can now insert the 5VDC power cable and flip the power switch labelled `SW700`, to the `on` position. 
+We should now have some glowing LEDs on the underside of the board indicating a sign of life. Next we need to press and hold the `SW400 POWER` push button for 1 or 2 seconds, 
+this starts the boot from the {{ $device.bootMedia }}. You should see a the red `POWER ON` LED light up when it starts booting.
+
+After a few seconds you should see your {{ $device.name }} appear on the resin.io dashboard in a configuring state. 
+If it still hasn't appeared after 1 or 2 minutes, double check your ethernet connection or that you entered the right wifi credentials at download time. 
+If you still can't get it online, contact us on support.
```

Comment:
We should have a doc on our support channels so we can link to it in cases like this yea?